docs/PINOUT.MD

```diff
@@ -0,0 +1,13 @@
+# PINOUT for digital temperature sensor DS18B20
+                                                NUCLEO PIN                SENSOR PIN
+                                                   PC8  ------------------->  DQ
+                                                   +5V  ------------------->  VDD
+                                                   GND  ------------------->  GND
```

Comment:
An arrow should not be in the current direction? In the other case, we should make two tips.

Reply:
fixed